cairo/tests/ethereum/cancun/test_fork.py

```diff
@@ -270,3 +401,173 @@ def test_get_last_256_block_hashes(self, cairo_run, blocks):
         cairo_result = cairo_run("get_last_256_block_hashes", chain)
 
         assert py_result == cairo_result
+
+    @given(data=apply_body_data())
+    def test_apply_body(
+        self,
+        cairo_run_py,
+        data,
+    ):
+        (
+            transactions,
+            state,
+            chain_id,
+            block_hashes,
+            coinbase,
+            block_number,
+            base_fee_per_gas,
+            block_gas_limit,
+            block_time,
+            prev_randao,
+            parent_beacon_block_root,
+            excess_blob_gas,
+        ) = data
+        withdrawals = ()  # Empty withdrawals for now
+
+        try:
+            # TODO: Use cairo_run and Rust CairoVM
+            cairo_state, cairo_result = cairo_run_py(
+                "apply_body",
+                state,
+                block_hashes,
+                coinbase,
+                block_number,
+                base_fee_per_gas,
+                block_gas_limit,
+                block_time,
+                prev_randao,
+                transactions,
+                chain_id,
+                withdrawals,
+                parent_beacon_block_root,
+                excess_blob_gas,
+            )
+        except Exception as e:
+            # If Cairo implementation raises an error, Python implementation should too
+            with strict_raises(type(e)):
+                apply_body(
+                    state,
+                    block_hashes,
+                    coinbase,
+                    block_number,
+                    base_fee_per_gas,
+                    block_gas_limit,
+                    block_time,
+                    prev_randao,
+                    transactions,
+                    chain_id,
+                    withdrawals,
```

Comment:
why splitting the data into each component, then give each arg to the function. Just use `*data`

Reply:
It's better to keep the explicit unpacking to ensure the arguments are passed in the correct order

Reply:
I'll use kwargs